feedback/filters.py

```diff
@@ -272,7 +281,7 @@ class FeedbackFilter(django_filters.FilterSet):
     order_by = OrderingFilter(label=_("Sort"),
                               choices=ORDER_BY_CHOICE,
                               initial=ORDER_BY_DEFAULT)
-
+    paginate_by = PaginateByFilter(label=_("Paginate by"))
```

Comment:
The string "Paginate by" doesn't exist yet in the localization file, so I would suggest updating that as well. You can find instructions for updating translations in the [a-plus repository documentation](https://github.com/apluslms/a-plus/tree/master/doc#updating-translations-for-finnish-and-english-versions) (although in MOOC-Jutut there is only the Finnish localization file, but the same command works).